docs/docs/getting-started/install.md

```diff
@@ -278,84 +278,89 @@ The built-in `AdministratorAccess` policy is a superset of these permissions.
 
 To [create a Constellation cluster](../workflows/create.md#the-create-step), you need the following permissions:
 
+# TODO should we refer to the file instead of manually copying the changes.. Nobody seems to update it
```

Comment:
I think the idea of having this copied over here is that the versioned docs still provide the correct values. For referencing the correct file, we'd need to point to an not-yet-existing release branch.

Reply:
But does everyone really think of updating it? Do we have any process? We could still refer to the main.tf file with the release tag.

Reply:
> I think the idea of having this copied over here is that the versioned docs still provide the correct values. For referencing the correct file, we'd need to point to an not-yet-existing release branch.

But does everyone really think of updating it? Do we have any process? We could still refer to the main.tf file with the release tag.